samples/central-login/index.html

```diff
@@ -63,9 +63,9 @@
       scope: '<Your OAuth Scopes>', // e.g. 'openid profile me.read'
         serverConfig: {
         baseUrl: '<Your AM URL>', // e.g. 'https://openam.example.com:9443/openam/'
-        timeout: '<Your Timeout (milliseconds)>' // 90000 or less
+        timeout: '<Your Timeout (milliseconds)>' // 4000 is a good number. (90000 or less)
       },
-      support: '<Browser Support>', // e.g. 'legacy' or 'modern'
+      support: '<Browser Support>', // e.g. 'legacy'
```

Comment:
Let's remove this property all together. It defaults to legacy internally, so it's optional.